internal/provider/datasource_directory_roles.go

```diff
@@ -93,6 +94,10 @@ __Further documentation:__
 							MarkdownDescription: "Shows whether the role can be modified or not.",
 							Computed:            true,
 						},
+						"app_name": schema.StringAttribute{
+							MarkdownDescription: "The name of the xsuaa application.",
```

Comment:
i'm not sure about the terminology. Is this really called an xsuaa application? What is meant is something like the name of the app as registered in xsuaa, right? I'm also aware of the xsuaaAppName but is this also the term used by customer's?

Reply:
I checked in the cockpit and adjusted according to the label description there. I also aligned that with the ID.

Reply:
ok, thanks.